.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ permissions:
   contents: read
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.client_payload.ref }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref }}
```

Comment:
Does this end up being the branch name or the commit sha? If it's the later it means the PR workflow would not be cancelled for new pushes, not the end of the world but not quite perfect :D

Reply:
it's the branch name
```
      "head": {
        "label": "timescale:adol/add-merge-gatekeeper",
        "ref": "adol/add-merge-gatekeeper",
```

I investigated with this action:

```
steps:
  - name: Dump GitHub context
  uses: crazy-max/ghaction-dump-context@v1
```